web/styles/utils.sass

```diff
@@ -0,0 +1,62 @@
+.fill
+  &-width
```

Comment:
wtf i didnt know you could do this

Reply:
There is a bit of a gotcha though: I don't think parent class traits propagate to child classes, so anything defined under `fill` needs to be applied using the `@extends` directive on each child class

Could be wrong on this though